### DISCUSSION TITLE
Placeholder Api for Geolocation

### DISCUSSION THREAD

**Discussion by XeonG** [EssentialsX/Essentials#4859] [Ideas and Feature Suggestions] [answered]
I noticed the stuff for essentials at placeholderapi.com/placeholders doesn't have anything for getting the players country? 

Even though essentials is one of the few plugins that has the GEO plugin for getting that info, none of it is exposed for playerholder api... which is a disappointment.

Could it be added thanks

**Comment by mdcfe** [comment, marked as answer]
We don't develop the PlaceholderAPI Essentials expansion. You can request this feature at the [PAPI expansion repository](https://github.com/placeholderapi/essentials-expansion).
